Source/Particles/WarpXParticleContainer.H

```diff
@@ -388,11 +389,13 @@ public:
     int getSpeciesId() const {return species_id;}
 
     ///
-    /// This returns the total charge for all the particles in this ParticleContainer.
-    /// This is needed when solving Poisson's equation with periodic boundary conditions.
+    /// This returns the total energy/weight/charge for all the particles in this ParticleContainer.
+    /// The total charge is needed when solving Poisson's equation with periodic boundary conditions.
     ///
-    amrex::ParticleReal sumParticleCharge(bool local = false);
-    amrex::ParticleReal sumParticleWeight(bool local = false);
+    std::pair<amrex::ParticleReal, amrex::ParticleReal> sumParticleWeightAndEnergy(bool local = false) const;
+    amrex::ParticleReal sumParticleCharge(bool local = false) const;
+    amrex::ParticleReal sumParticleWeight(bool local = false) const;
+    amrex::ParticleReal sumParticleEnergy(bool local = false) const;
```

Comment:
Suggesting to leave a white space between the function's name and the list of arguments, following https://warpx.readthedocs.io/en/latest/developers/contributing.html#style-and-conventions:
```suggestion
    std::pair<amrex::ParticleReal, amrex::ParticleReal> sumParticleWeightAndEnergy (bool local = false) const;
    amrex::ParticleReal sumParticleCharge (bool local = false) const;
    amrex::ParticleReal sumParticleWeight (bool local = false) const;
    amrex::ParticleReal sumParticleEnergy (bool local = false) const;
```